### PR TITLE
Donald

### PR DESCRIPTION
**Pull request by DDRIV**
Finished event pathway

### CODE REVIEW

File: src/test/java/com/revature/RevAssure/service/EventServiceTest.java [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+import org.junit.jupiter.api.Test;
+import org.mockito.Mockito;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.Mockito.when;
+
```
Comment (cptvictor) [inline src/test/java/com/revature/RevAssure/service/EventServiceTest.java:18]: add @SpringBootTest

File: src/test/java/com/revature/RevAssure/service/EventServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+class EventServiceTest {
+    private CurriculumRepository curriculumRepository;
```
Comment (cptvictor) [inline src/test/java/com/revature/RevAssure/service/EventServiceTest.java:20]: add @MockBean

File: src/test/java/com/revature/RevAssure/service/EventServiceTest.java [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+    private EventRepository eventRepository;
+    private EventService eventService;
```
Comment (cptvictor) [inline src/test/java/com/revature/RevAssure/service/EventServiceTest.java:22]: add @Autowired

**Review by AlwynZ — APPROVED** [review]
lgtm

File: src/test/java/com/revature/RevAssure/service/EventServiceTest.java [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+        when(eventRepository.save(event)).thenReturn(event);
+
+        Event ev = eventService.updateEvent(event);
+
+        assertEquals(event,ev);
+    }
+
+    @Test
+    void deleteEventSuccessfullyTest() {
+        when(eventRepository.getById(1)).thenReturn(event);
```
Comment (cptvictor) [inline src/test/java/com/revature/RevAssure/service/EventServiceTest.java:91]: delete should not return, verify lines run (see/ask Amit)

**Review by cptvictor — CHANGES_REQUESTED** [review]
Please make changes to your test class

File: src/test/java/com/revature/RevAssure/service/EventServiceTest.java (hunk 1 above)
Comment (DDRIV) [reply src/test/java/com/revature/RevAssure/service/EventServiceTest.java:20]: completed

File: src/test/java/com/revature/RevAssure/service/EventServiceTest.java (hunk 2 above)
Comment (DDRIV) [reply src/test/java/com/revature/RevAssure/service/EventServiceTest.java:24]: completed

File: src/test/java/com/revature/RevAssure/service/EventServiceTest.java (hunk 3 above)
Comment (DDRIV) [reply src/test/java/com/revature/RevAssure/service/EventServiceTest.java:28]: completed

File: src/test/java/com/revature/RevAssure/service/EventServiceTest.java (hunk 4 above)
Comment (DDRIV) [reply src/test/java/com/revature/RevAssure/service/EventServiceTest.java:93]: delete is not returning, that's the getById method call inside the delete method

**Review by AlwynZ — APPROVED** [review]
lgtm

**Review by AlwynZ — APPROVED** [review]
lgtm

**Pull request merged**